src/example_code/Integer ornare.java

```diff
@@ -0,0 +1,7 @@
+// BEGIN FRAGMENT: Integer ornare
+class IntegerOrnare {
+    public static void main(String[] args) {
+        System.out.println("Integer ornare"); 
```

Comment:
Bad EOF. Relevant for other `example_code`s

<img width="176" alt="image" src="https://user-images.githubusercontent.com/43530070/190311569-3a9c809b-7db2-4e4d-8425-bc68b4eb31ca.png">